components/checkbox/Group.tsx

```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 import shallowEqual from 'shallowequal';
 import Checkbox from './Checkbox';
 
-export type CheckboxValueType = string | number;
```

Comment:
interesting, lint complains: `Type 'CheckboxValueType' is not assignable to type 'string | number | undefined'.`